sycl/doc/design/CommandGraph.md

```diff
@@ -0,0 +1,62 @@
+# Command Graph Extension
+
+This document describes the implementation design of the 
+[SYCL Graph Extension](https://github.com/intel/llvm/pull/5626).
+
+A related presentation can be found 
+[here](https://www.youtube.com/watch?v=aOTAmyr04rM).
+
+## Requirements
+
+An efficient implementation of a lazy command graph execution and its replay
+requires extensions to the PI layer. Such an extension is command buffers,
+where a command-buffer object represents a series of operations to be enqueued
+to the backend device and their dependencies. A single command graph can be
+partitioned into more than one PI command-buffer by the runtime.
+We distinguish between backends that support command buffer extensions and
+those that do not. Currently command buffer extensions are only supported by
+Level Zero. All other backends would fall back to an emulation mode.
```

Comment:
"All other backends would fall back to an emulation mode, **or not be reported as supported**"

Reply:
done